loadgen/bindings/c_api.h

```diff
@@ -40,11 +40,15 @@ typedef void (*IssueQueryCallback)(ClientData, const QuerySample*, size_t);
 typedef void (*FlushQueriesCallback)();
 typedef void (*ReportLatencyResultsCallback)(ClientData, const int64_t*,
                                              size_t);
+typedef void (*EndOnDeviceCallback)(ClientData, QuerySampleResponse*);
 
 /// \brief SUT calls this function to report query result back to loadgen
 void QuerySamplesComplete(QuerySampleResponse* responses,
                           size_t response_count);
 
+void QuerySamplesCompleteEndOnDevice(QuerySampleResponse* responses,
```

Comment:
Need new function in C API because C doesn't support default arguments or function overloading.